apps/session-manager/env.example

```diff
@@ -1,3 +1,12 @@
+################
+# AppInsinghts #
+################
+APPLICATIONINSIGHTS_CONNECTION_STRING=""
```

Comment:
is this `APPINSIGHTS_INSTRUMENTATIONKEY`?

Reply:
`APPINSIGHTS_INSTRUMENTATIONKEY` is now deprecated